onnx/version_converter/helper.cc

```diff
@@ -36,20 +36,15 @@ void assert_numpy_multibroadcastable(
     const std::vector<Dimension>& input1_sizes,
     const std::vector<Dimension>& input2_sizes) {
   // Generalize above for multibroadcastable case
-  const std::vector<Dimension>* A_ptr;
-  const std::vector<Dimension>* B_ptr;
-  int A;
-  int B;
+  const std::vector<Dimension>* A_ptr = &input1_sizes;
+  const std::vector<Dimension>* B_ptr = &input2_sizes;
+  int A = 1;
+  int B = 2;
   if (input1_sizes.size() < input2_sizes.size()) {
     A_ptr = &input2_sizes;
```

Comment:
Instead, declare in both branches ?

Reply:
The old behavior seems to declare first and initialize in the branches. If we declare in both branches we won’t have the risk of uninitialized variables this is solving. Please correct me if I am wrong

Reply:
Can't declare in both branches because the variables are used below the branches.